docs/openstack-deploy-cli.md

```diff
@@ -0,0 +1,47 @@
+# Openstack Deploying the Command Line Tools
+
+Before we can get started we need to install a few things.
+
+#### Installing Python
+
+Installed by default on Mac OS X.
+
+Many Linux distributions provide packages to make setuptools easy to install. Search your package manager for setuptools to find an installation package. If you cannot find one, download the setuptools package directly from https://pypi.python.org/pypi/setuptools.
+
+The recommended way to install setuptools on Microsoft Windows is to follow the documentation provided on the setuptools website (https://pypi.python.org/pypi/setuptools).
+
+
+#### Installing pip
+
+MacOS
+
+``` shell
+easy_install pip
```

Comment:
For macos I think we should reference brew or pyenv? Something that doesn't carry the risk of hurting a users default python environment.

Reply:
Updated